src/config.hpp.in

```diff
@@ -20,25 +20,25 @@
 #ifndef ONEMKL_CONFIG_H
 #define ONEMKL_CONFIG_H
 
-#cmakedefine ENABLE_CUBLAS_BACKEND
-#cmakedefine ENABLE_CUFFT_BACKEND
-#cmakedefine ENABLE_CURAND_BACKEND
-#cmakedefine ENABLE_CUSOLVER_BACKEND
-#cmakedefine ENABLE_MKLCPU_BACKEND
-#cmakedefine ENABLE_MKLGPU_BACKEND
-#cmakedefine ENABLE_NETLIB_BACKEND
-#cmakedefine ENABLE_PORTBLAS_BACKEND
-#cmakedefine ENABLE_PORTBLAS_BACKEND_AMD_GPU
-#cmakedefine ENABLE_PORTBLAS_BACKEND_INTEL_CPU
-#cmakedefine ENABLE_PORTBLAS_BACKEND_INTEL_GPU
-#cmakedefine ENABLE_PORTBLAS_BACKEND_NVIDIA_GPU
-#cmakedefine ENABLE_PORTFFT_BACKEND
-#cmakedefine ENABLE_ROCBLAS_BACKEND
-#cmakedefine ENABLE_ROCFFT_BACKEND
-#cmakedefine ENABLE_ROCRAND_BACKEND
-#cmakedefine ENABLE_ROCSOLVER_BACKEND
-#cmakedefine BUILD_SHARED_LIBS
-#cmakedefine REF_BLAS_LIBNAME "@REF_BLAS_LIBNAME@"
-#cmakedefine REF_CBLAS_LIBNAME "@REF_CBLAS_LIBNAME@"
+#cmakedefine ONEAPI_ONEMKL_ENABLE_CUBLAS_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_CUFFT_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_CURAND_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_CUSOLVER_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_MKLCPU_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_MKLGPU_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_NETLIB_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_PORTBLAS_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_PORTBLAS_BACKEND_AMD_GPU
+#cmakedefine ONEAPI_ONEMKL_ENABLE_PORTBLAS_BACKEND_INTEL_CPU
+#cmakedefine ONEAPI_ONEMKL_ENABLE_PORTBLAS_BACKEND_INTEL_GPU
+#cmakedefine ONEAPI_ONEMKL_ENABLE_PORTBLAS_BACKEND_NVIDIA_GPU
+#cmakedefine ONEAPI_ONEMKL_ENABLE_PORTFFT_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_ROCBLAS_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_ROCFFT_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_ROCRAND_BACKEND
+#cmakedefine ONEAPI_ONEMKL_ENABLE_ROCSOLVER_BACKEND
+#cmakedefine ONEAPI_ONEMKL_BUILD_SHARED_LIBS
+#cmakedefine ONEAPI_ONEMKL_REF_BLAS_LIBNAME "@ONEAPI_ONEMKL_REF_BLAS_LIBNAME@"
+#cmakedefine ONEAPI_ONEMKL_REF_CBLAS_LIBNAME "@ONEAPI_ONEMKL_REF_CBLAS_LIBNAME@"
```

Comment:
These two can be removed now

Reply:
Addressed while rebasing to include this change